src/libraries/Microsoft.Extensions.Configuration.Xml/tests/ConfigurationProviderXmlTest.cs

```diff
@@ -33,11 +43,8 @@ public override void Has_debug_view()
     Key3=Value123 ({providerTag})
     Key3a:
       0=ArrayValue0 ({providerTag})
-        Name=0 ({providerTag})
```

Comment:
Why would this behavior need to change?

Reply:
The debug view test (ab)used the "Name" attribute for arrays. 
This is no longer necessary since the new implementation can handle arrays without having to use the "Name" attribute.